src/components/layouts/MainLayout.tsx

```diff
@@ -1,13 +1,18 @@
 import type { ReactNode } from "react";
 
+import Footer from "@/components/Footer";
+
 interface Props {
   children: ReactNode | ReactNode[];
 }
 
 export default function MainLayout({ children }: Props) {
   return (
-    <main className="flex min-h-screen flex-col items-center justify-between p-24">
-      {children}
-    </main>
+    <>
```

Comment:
nit: is this JSX fragment needed?

Reply:
This fragment is needed because JSX expressions must have one parent element.